entities.py

```diff
@@ -0,0 +1,315 @@
+"""
+To-do:
+
+Sprites for different entities
+
+
+Notes:
+
+Removed need for global variables. Now initializing player requires x, y, and grid, and each monster needs x, y, player and grid.
+It's a bit more of a pain to initialize (more variables required) but we can change stuff and feed it in, instead of them grabbing the global 
+    value for 'player' and 'grid'.
+"""
+
+from random import randint
+
+
+
+"""Changes:"""
+
+# added 'effect' -> basically, being poisoned or something. It's a dictionary, with True or False for whether the effect is active. We can add in what the effect actually does eventually. 
+# also added the action of being effected, and can call Entity.active_effects() to get a list of all active effects. 
+
+# added in items, and made it so items can be added to the inventory and used, along with some fun text descriptions for their appearence and usage.
+# made potion class, for easy potion making (writing all those descriptions is a pain, this way you just have to put what they do)
+
+# also added max health, so that when the player uses potions they can't go past their max health. 
+
+# ex: Anna.effected('poisoned') -> "You've been poisoned!" -> Anna.effect['poisoned'] = True -> Anna.active_effects = ['poisoned'] -> cure_potion = Potion('cure','poisoned') 
+# 	  cure_potion.pickup(Anna) -> (fun flair text) -> Anna.inventory = {'Weird Blue Potion' , 1} -> cure_potion.use(Anna) -> (fun cured flair text) -> Anna.active_effects = []
+
+
+class Entity(object):
+    def __init__(self, grid, direction="U", health=30, maxhealth=30, speed=1, accuracy=2, flatDamage = 2, damageRange=2, damageMod=2, vision=3, armor=10, phasing = False, name = None, effect = dict()):
```

Comment:
Take out all the defaults on here (except for phasing). There's no reason to have them, since it will probably vary almost completely from monster to monster. Just have each of the monster types pass it the specific you want.